lochness/redcap/__init__.py

```diff
@@ -280,7 +280,7 @@ def sync(Lochness, subject, dry=False):
                 if not os.path.exists(dst):
                     logger.debug(f'saving {dst}')
                     lochness.atomic_write(dst, content)
-                    process_and_copy_db(Lochness, subject, dst, proc_dst)
+                    # process_and_copy_db(Lochness, subject, dst, proc_dst)
```

Comment:
Sylvain has plans to manage a separate repo for PII processing. I'm testing this branch with Z today, so commenting this line out, to quickly prevent PII processing from happening.